modules/nf-core/gatk4/createsequencedictionary/tests/main.nf.test

```diff
@@ -0,0 +1,57 @@
+nextflow_process {
+
+    name "Test Process GATK4_CREATESEQUENCEDICTIONARY"
+    script "../main.nf"
+    process "GATK4_CREATESEQUENCEDICTIONARY"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "gatk4"
+    tag "gatk4/createsequencedictionary"
+
+    test("sarscov2 - fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test' ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/genome.fasta', checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out.versions).match("versions") },
+                { assert snapshot(file(process.out.dict.get(0).get(1)).name).match() }
```

Comment:
while not snapshotting the file?

Reply:
I changed it to read the lines. The md5sum changes between docker and conda for me.

Reply:
I see.
The old pytest was using the md5sum which is why I was suggesting this approach

Reply:
I tried it again, maybe the CI works? I just ran into weird stuff on my computer